inc/saiobject.h

```diff
@@ -58,37 +58,43 @@ typedef struct _sai_object_key_t
 /**
```

Comment:
@AshokDaparthi :  as a follow up on @atitjain's question just for my understanding what is the motivation that there the following will be different for various switch-ids in the system : 
1. max attributes per object type per switchid (heterogeneous ASIC supported ? )
2. number of objects per object type per switchid (asymmetric type of programming done across multiple ASICs?)